buildenv/jenkins/ARTIFACTORY.md

```diff
@@ -146,18 +146,18 @@ server {
     chunked_transfer_encoding on;
     client_max_body_size 0;
     location / {
-    proxy_read_timeout  900;
-    proxy_pass_header   Server;
-    proxy_cookie_path   ~*^/.* /;
-    if ( $request_uri ~ ^/artifactory/(.*)$ ) {
-        proxy_pass          http://localhost:8081/artifactory/$1;
-    }
-    proxy_pass         http://localhost:8081/artifactory/;
-    proxy_set_header   Host $host;
-    proxy_set_header    X-Forwarded-Port  $server_port;
-    proxy_set_header    X-Forwarded-Proto $http_x_forwarded_proto;
-    proxy_set_header    Host              $http_host;
-    proxy_set_header    X-Forwarded-For   $proxy_add_x_forwarded_for;
+        proxy_read_timeout  900;
+        proxy_pass_header   Server;
+        proxy_cookie_path   ~*^/.* /;
+        if ($request_uri ~ ^/artifactory/(.*)$) {
+            proxy_pass      http://localhost:8081/artifactory/$1;
+        }
+        proxy_pass          http://localhost:8081/artifactory/;
+        proxy_set_header    Host $host;
+        proxy_set_header    X-Forwarded-Port  $server_port;
+        proxy_set_header    X-Forwarded-Proto $http_x_forwarded_proto;
+        proxy_set_header    Host              $http_host;
+        proxy_set_header    X-Forwarded-For   $proxy_add_x_forwarded_for;
```

Comment:
I'm inclined not to adjust this as it's a copy/paste of a config file for the server. It shouldn't be a problem but we would have copied it from some other wiki/tutorial like this.

Reply:
This mostly is just indenting like (similar content at) line 85.